resources/public/contracts/src/Offering.sol

```diff
@@ -170,16 +170,17 @@ contract Offering {
 
     /**
     * @dev Function to actually do ENS transfer
-    * Top level names should be transferred via registrar, so deed is transferred too
+    * Top level names should be transferred via registrar, so that registration is transferred too
     * @param _newOwner address New owner of ENS name
     */
     function doTransferOwnership(address payable _newOwner)
         private
     {
+        ens.setOwner(offering.node, _newOwner);
```

Comment:
Before, the registry ownership was transferred in both cases as well, but one of them was masked inside the `HashRegistrar.transfer` function. So there is not a change in this respect.

If you keep the registration token ownership, transferring registry ownership only is sort of worthless, since anyone owning the registration token may `reclaim` registry ownership at any time. So you need to transfer the registration (token).

You can decide, whether you transfer registry ownership while you're still holding the registration, or you just transfer the registration and let the other side reclaim. I chose the former to make life easier for receivers (which could be you).

Reply:
> Before, the registry ownership was transferred in both cases as well, but one of them was masked inside the HashRegistrar.transfer function. So there is not a change in this respect.

Thanks, this was the part that I missed :) 

> You can decide, whether you transfer registry ownership while you're still holding the registration.

For non TLD of `.eth` there is just ownership and that's trivial case - just set the new owner.
For TLDs: If I just transfer ownership, but I still own the registration I can just `reclaim` it back right? In both cases I need to transfer the registration. The question is whether to transfer ownership or not as well. ANd I agree with your choice to transfer it as well and make it easier for receivers. Is my understanding correct?

Reply:
Your understanding is same as mine, which is hopefully also correct ;)